database/things/http.pldb

```diff
@@ -73,6 +73,8 @@ leachim6 HTTP
   httpd = socketserver.TCPServer(('', 8000), Handler)
   httpd.serve_forever()
   
+indeedJobs http developer
+ 2022 771996 
```

Comment:
How do you determine the job requirements of a "http developer" ? It seems to be a very broad category.

Reply:
Here was looking at jobs that have HTTP as a requirement

Reply:
its a very very broad category